cpp/src/parquet/properties.h

```diff
@@ -452,19 +452,39 @@ class PARQUET_EXPORT WriterProperties {
       return this->disable_statistics(path->ToDotString());
     }
 
-    /// Enable integer type to annotate decimal type as below:
-    ///   int32: 1 <= precision <= 9
-    ///   int64: 10 <= precision <= 18
-    /// Default disabled.
-    Builder* enable_integer_annotate_decimal() {
-      integer_annotate_decimal_ = true;
+    /// Enable decimal logical type with 1 <= precision <= 18 to be stored as
+    /// integer physical type.
```

Comment:
```suggestion
    /// Allow decimals with precision <= 18 to be stored as integers.
```

Reply:
We can omit `1 <= precision` since that's obvious.

Reply:
I'd prefer to keep 1 <= precision <= 18 to be consistent with the specs. Otherwise we may infer that precision <= 0 (including negative precisions) also apply this.

Reply:
Do negative precision decimals exist?

Reply:
I don't think negative precision decimals exist, but keeping `1 <= precision` doesn't hurt.